src/tools/prun/prun.1.md

```diff
@@ -443,34 +281,39 @@ The parser for the `-x` option is not very sophisticated; it does not
 even understand quoted values. Users are advised to set variables in the
 environment, and then use `-x` to export (not define) them.
 
-Setting MCA parameters:
+## MCA Parameters
 
-`-gpmca, --gpmca <key> <value>`
+Setting MCA parameters take a few different forms depending the target
+project for the parameter. For example, MCA parameters targeting OpenPMIx
+will contain the string `pmix` in their name, and MCA parameters targeting
+PRTE will contain the string `prte` in their name. See the "MCA" section,
+below, for finer details on the MCA.
 
-:   Pass global MCA parameters that are applicable to all contexts.
+`--gpmixmca <key> <value>`
+
+:   Pass global PMIx MCA parameters that are applicable to all contexts.
     `<key>` is the parameter name; `<value>` is the parameter value.
 
-`-pmca, --pmca <key> <value>`
+`--mca <key> <value>`
 
 :   Send arguments to various MCA modules. See the "MCA" section,
     below.
 
-`-am <arg0>`
+`--pmixmca <key> <value>`
 
-:   Aggregate MCA parameter set file list.
+:   Send arguments to various PMIx MCA modules. See the "MCA" section,
```

Comment:
Is this supposed to be the pathname to a file?

Reply:
Yep. I added a clarification.
```
:   Aggregate PMIx MCA parameter set file list. The `arg0` argument is a
    comma-separated list of tuning files. Each file containing MCA parameter
    sets for this application context.
```